Jenkinsfile

```diff
@@ -14,116 +14,8 @@
-                                }
-                            }
-                        }
-
-                        stage('Docker Push') {
-                            when { expression { edgex.isReleaseStream() } }
-                            steps {
-                                script {
-                                    docker.withRegistry("https://${env.DOCKER_REGISTRY}:10004") {
-                                        image_amd64.push(env.GIT_COMMIT)
-                                        image_amd64.push('1.1.0')
-                                    }
-                                }
-                            }
-                        }
-                    }
-                }
-
-                stage('arm64') {
-                    agent {
-                        label 'ubuntu18.04-docker-arm64-4c-2g'
-                    }
-                    stages {
-                        stage('Docker Build') {
-                            steps {
-                                edgeXDockerLogin(settingsFile: env.MVN_SETTINGS)
-
-                                script {
-                                    image_arm64 = docker.build('docker-edgex-consul-arm64', '-f Dockerfile .')
-                                }
-                            }
-                        }
-
-                        stage('Docker Push') {
-                            when { expression { edgex.isReleaseStream() } }
-                            steps {
-                                script {
-                                    docker.withRegistry("https://${env.DOCKER_REGISTRY}:10004") {
-                                        image_arm64.push(env.GIT_COMMIT)
-                                        image_arm64.push('1.1.0')
-                                    }
-                                }
-                            }
-                        }
-                    }
-                }
-            }
-        }
-
-        stage('Clair Image Scan') {
-            when { expression { edgex.isReleaseStream() } }
-            steps {
-                edgeXClair("${env.DOCKER_REGISTRY}:10004/docker-edgex-consul:1.1.0")
-                edgeXClair("${env.DOCKER_REGISTRY}:10004/docker-edgex-consul-arm64:1.1.0")
-            }
-        }
-    }
-
-    post {
-        always {
-            edgeXInfraPublish()
-        }
-    }
-}
-
-def loadGlobalLibrary(branch = '*/master') {
-    library(identifier: 'edgex-global-pipelines@master', 
-        retriever: legacySCM([
-            $class: 'GitSCM',
-            userRemoteConfigs: [[url: 'https://github.com/edgexfoundry/edgex-global-pipelines.git']],
-            branches: [[name: branch]],
-            doGenerateSubmoduleConfigurations: false,
-            extensions: [[
-                $class: 'SubmoduleOption',
-                recursiveSubmodules: true,
-            ]]]
-        )
-    ) _
-}
\ No newline at end of file
+edgeXBuildDocker (
+    project: 'docker-edgex-consul',
```

Comment:
If both the project and the dockerImageName are the same you can leave out the dockerImageName